start-as-experiment.md

```diff
@@ -0,0 +1,81 @@
+# Title
+
+Start as an Experiment
+
+# Problem
+
+An InnerSource initiative is considered but not started because management is 
+unsure about its outcome and, as a result, is not willing to commit to an 
+investment.
+
+# Context
+
+The majority of software development projects in the company are distributed
+globally. The company is considering to leverage InnerSource to make the
+required collaboration more efficient. However, most managers are not familiar
+with the Open Source working model and are instead accustomed to hierarchical,
+top-down control style management. The idea of InnerSource is very popular with
+software developers in the company, not the least because many developers use
+or are actively developing Open Source software.
+
+# Forces
+
+- Managers will want to validate the claims of improved collaboration through
```

Comment:
From 7/18/17 patterns meeting: Engineering-led inner source often does not scale to the wider organization because engineers lack a marketing skillset and upward/sideways org traversal.

Are we able to weave this interesting comment/observation in anywhere?

Reply:
Is this comment/observation directly related to the thrust and intent of this pattern?

Reply:
I think this is not directly related, @nyeates. This force was geared towards the desire of mgmt. to validate a new idea before they sign off on it. So it's more related to KPIs and measurements than marketing skills.